cibuild.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
```

Comment:
Maybe add `set -x` and `set -e` options to aid CI log/debug?

Reply:
That's right. set -x and -e would be good to debug.

Reply:
done